### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Dmytro-Kretov**
- [DEMO LINK](https://Dmytro-Kretov.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://Dmytro-Kretov.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

**Review by ArturErmolenko — APPROVED** [review]
Well done 👍

**Review by aliceblunt8 — CHANGES_REQUESTED** [review]
the difference with design is huge, please fix

Screenshot 2022-06-14 at 16 27 16

**Comment by Dmytro-Kretov** [comment]
I guess this time its much better. 

but could I ask you for a favor, next time on requesting changes I would like to ask you to give some hints on where to make changes, cuz its really hard sometimes to search for what's wrong

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,44 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img class="card__image" 
+          src="./images/imac.jpeg" 
+          alt="MAC"
+      >
```
Comment (aliceblunt8) [inline src/index.html:18]:
```suggestion
      <img
        class="card__image" 
        src="./images/imac.jpeg" 
        alt="MAC"
      >
```

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,44 @@
...
+      <div class="card__price">
+        <p class="card__price-price">Цена: </p>
+        <p class="card__price-sum">69 999 грн</p>
+      </div>
+      <a href="#" 
+      data-qa="hover"
+      class="card__buy-link"
+      >
+      Купить
+      </a>
```
Comment (aliceblunt8) [inline src/index.html:41]:
```suggestion
      <a
        href="#" 
        data-qa="hover"
        class="card__buy-link"
      >
        Купить
      </a>
```

**Review by aliceblunt8 — APPROVED** [review]

**Pull request closed without merging**